core/src/main/java/io/questdb/TelemetrySystemEvent.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb;
+
+public final class TelemetrySystemEvent {
+    public static final short APPLIED_TXN = 103;
```

Comment:
Names are not very intuitive
```suggestion
    public static final short WAL_APPLY_WAIT_QUEUE = 103;
```

Reply:
this event is not about the queue.
it is logged when the txn is applied to the table.
`APPLIED_TXN` = job has applied the txn to the table.

event come like this:
`APPLY_TXN_DATA` - apply job applies data transaction (this logs the latency added by the queue)
`APPLIED_TXN` - apply job applied the transaction (this logs the latency caused by the commit)
or like this:
`APPLY_STRUCTURE_CHANGE` - apply job applies structure change transaction (this logs the latency added by the queue)
`APPLIED_TXN` - apply job applied the transaction (this logs the latency caused by the alter)

`WAL_APPLY` was not in the name because there is an origin which tells that it is the apply job doing it.
with the removal of the `origin` field it makes sense to add it as a prefix to the events.

Reply:
I am confused of the timing of the events, see my comments in `ApplyWal2TableJob`

Reply:
Fixed in https://github.com/questdb/questdb/pull/2930/commits/6ecb9fddca771bfe59923c88c596b964b9e01879